vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridMultiSelectionColumnPageIT.java

```diff
@@ -66,6 +66,33 @@ public void selectAllCheckbox() {
                 message.getText());
     }
 
+    @Test
+    public void selectCheckboxMultiSelectionMode() {
+        open();
+
+        // Test switch selection mode from single to multi mode before adding the grid to DOM
+        WebElement gridSelectionMode = findElement(By.id("in-testing-multi-selection-mode-grid"));
+        WebElement selectAllCheckbox_selectionMode = gridSelectionMode
+                .findElement(By.id("selectAllCheckbox"));
+        WebElement message_selectionMode = findElement(By.id("selected-item-count"));
+        Assert.assertTrue(String.valueOf(selectAllCheckbox_selectionMode.isDisplayed()), true);
```

Comment:
`Assert.assertTrue(...,true)` is pointless, it will always pass.

Reply:
done